### PR TITLE
Add get_task_status method to Algolia::Index class.

### PR DESCRIPTION
**Pull request by srodman7689**
After looking through the source of this library I realized that there wasn't a way to just check the status of a task instead of waiting for a task to finish. So I decided to create a pull request in the hopes that you guys would merge it and add this functionality. I could really use it.

### CODE REVIEW

**Comment by srodman7689** [comment]
I have created an issue in case you guys don't want to go this way with adding the functionality to get a tasks status here https://github.com/algolia/algoliasearch-client-ruby/issues/199

**Comment by redox** [comment]
> After looking through the source of this library I realized that there wasn't a way to just check the status of a task instead of waiting for a task to finish. So I decided to create a pull request in the hopes that you guys would merge it and add this functionality. I could really use it.

Awesome @srodman7689 👍  Thank you so much for contributing!

File: spec/client_spec.rb [hunk 1]
```diff
@@ -693,7 +693,7 @@ def wait_global_key_missing(key)
 
   it 'Check attributes get_task_status' do
     task = @index.add_object!({ :name => "John Doe", :email => "john@doe.org" }, "1")
-    status = @index.get_task_status(task["taskID"])
+    status = @index.get_task_status(task["objectID"])
```
Comment (redox) [inline spec/client_spec.rb]: This was actually correct, we what to get the status of the `taskID` not the `objectID`.

**Comment by redox** [comment]
Travis will fail because of the encrypted API keys we're using. I've run it locally 👍

**Pull request merged**